services/src/test/scala/cromwell/pipeline/service/ProjectFileServiceTest.scala

```diff
@@ -28,7 +27,8 @@ class ProjectFileServiceTest extends AsyncWordSpec with Matchers with MockitoSug
     "validateFile" should {
       "return valid message to valid file" taggedAs Service in {
         val request = ProjectFileContent(correctWdl)
-        when(womTool.validate(correctWdl)).thenReturn(Right(any[WomBundle]))
+        val dummyBundle = WomBundle(None, Map.empty, Map.empty, Set.empty)
```

Comment:
unrelated change?

Reply:
Nope. This test also has a problem with a misplaced matcher `any[Foo]` and might fail eventually.